server/models/room.ts

```diff
@@ -18,6 +18,7 @@ export interface RoomAttributes {
 	"role-mod": Array<number>;
 	"role-trusted": Array<number>;
 	"autoSkipSegments": boolean;
+	"autoSkipSegmentCategories": SegmentCategories;
```

Comment:
I would prefer if this replaced the boolean column instead of getting added alongside it. It'll make the db migration a little bit more complex, but I think it'll be worth it for the reduced complexity in the logic.